ConsoleAppHomeworkText/ConsoleAppHomeworkText/Program.cs

```diff
@@ -0,0 +1,215 @@
+            Console.WriteLine("\t3. Search by name");
+            Console.WriteLine("\t4. Update number");
+            Console.WriteLine("\t0. Exit");
+
+            ConsoleKeyInfo ck = Console.ReadKey();
+
+            switch (ck.Key)
+            {
+                case ConsoleKey.D1:
+                case ConsoleKey.NumPad1:
+                    ShowAllNumbers();
+                    break;
+                case ConsoleKey.D2:
+                case ConsoleKey.NumPad2:
+                    CreatePhoneNumber();
+                    break;
+                case ConsoleKey.D3:
+                case ConsoleKey.NumPad3:
+                    SearchByName();
+                    break;
+                case ConsoleKey.D4:
+                case ConsoleKey.NumPad4:
+                    UpdateNumber();
+                    break;
+                case ConsoleKey.D0:
+                case ConsoleKey.NumPad0:
+                    Exit();
+                    break;
+            }
+        }
+        private static void UpdateNumber()
+        {
+            Console.Clear();
+            Console.WriteLine("Please insert a new number");
+            string updatedNumber = Console.ReadLine();
+            Console.WriteLine("Search condition - if first/last name contains search term you will see it");
+
+            Console.WriteLine("Enter Search Term...");
+            var searchTerm = Console.ReadLine();
+            bool find = false;
+            var lines = ReadPhoneBook();
+            var linesUpdated = "FirstName,LastName,PhoneNumber\n";
+            for (int i = 0; i < lines.Length; i++)
+            {
+                (string firstName, string lastName, string phoneNumber) current = lines[i];
+                if (current.firstName.Contains(searchTerm, StringComparison.OrdinalIgnoreCase) ||
+                    current.lastName.Contains(searchTerm, StringComparison.OrdinalIgnoreCase))
+                {
+                    current.phoneNumber = updatedNumber;
+                    Console.WriteLine($"Number for {current.firstName} {current.lastName} successfully changed");
+                }
+                linesUpdated += $"{current.firstName},{current.lastName},{current.phoneNumber} \n";
+            }
+
+            File.WriteAllText(Filename, linesUpdated);
+
+            if (!find)
+            {
+                Console.WriteLine("Not such users");
+                Wait();
+                return;
+            }
+
+
+        }
+        //додати можливість оновлювати номер телефону за іменем/прізвищем користувача(ім'я/прізвище вводити повністю, але реєстр не важливий)
+        private static void SearchByNumber()
+        {
+            Console.Clear();
+            var search = Console.ReadLine();
+
+            bool found = false;
+            foreach (var (firstName, lastName, phoneNumber) in ReadPhoneBook())
+            {
+                if (phoneNumber.Contains(search, StringComparison.OrdinalIgnoreCase))
```

Comment:
not critical, but you do not need StringComparison.OrdinalIgnoreCase when searching for number